### DISCUSSION TITLE
point_0703

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#258] [Roy]
"20200609T163839_20200609T164248_T16TDM"
![image](https://user-images.githubusercontent.com/54947195/147531789-a016fd41-3164-4b6a-9eae-260577a28718.png)

"20200525T163901_20200525T165159_T16TDM"
![image](https://user-images.githubusercontent.com/54947195/147531887-1d5fea5c-e2fe-4d28-a8c8-7857f94ffef5.png)
![image](https://user-images.githubusercontent.com/54947195/147532014-b804fdc3-d91b-4554-be27-29ed631582fe.png)

"20200410T163839_20200410T164710_T16TDM"
![image](https://user-images.githubusercontent.com/54947195/147532125-e3214891-4f6d-4207-9834-285e987a7976.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20200609T163839_20200609T164248_T16TDM"
![Captura](https://user-images.githubusercontent.com/86502862/153074327-f9d17a91-244c-4e3f-ba68-8a491177537b.JPG)
